samples/tensorflow_lite/config.json

```diff
@@ -0,0 +1,14 @@
+{
+    "Debug": 1,
+    "StackMemSize": "800k",
+    "NumUserThreads": 8,
```

Comment:
We don't need NumUserThreads and StackMemSize anymore.

Reply:
Removed the unused elements.